mixer/pkg/runtime2/runtime.go

```diff
@@ -141,6 +141,9 @@ func (c *Runtime) StopListening() {
 	if c.shutdown != nil {
 		c.shutdown <- struct{}{}
 		c.shutdown = nil
+		if err := c.store.Close(); err != nil {
+			log.Errorf("Failed on close: %v", err)
```

Comment:
Can you improve the error message (i.e.  "store.Close failed during runtime shutdown: %v")

Reply:
Done